Classes/Common/DocumentList.php

```diff
@@ -728,7 +733,10 @@ public function __construct(array $elements = [], array $metadata = [])
             // Add initial set of elements to the list.
             $this->elements = $elements;
         }
+
         $this->count = count($this->elements);
+
+        $this->logger = GeneralUtility::makeInstance(LogManager::class)->getLogger(__CLASS__);
```

Comment:
This will fail as LogManager is unknow in this class.

Is this still necessary with LoggerAwareInterface?

Reply:
No, I have removed it.